devicedefender/CMakeLists.txt

```diff
@@ -111,6 +114,12 @@ else()
     set(TARGET_DIR "static")
 endif()
 
+include(CMakePackageConfigHelpers)
+write_basic_package_version_file(
+        "${CMAKE_CURRENT_BINARY_DIR}/iotdevicedefender-cpp-config-version.cmake"
+        COMPATIBILITY ExactVersion
```

Comment:
Since the IoT SDK is `1.x` wouldn't we want compatibility to be `SameMajorVersion`?